addon/templates/components/frost-modal-input.hbs

```diff
@@ -33,11 +33,15 @@
     minScrollbarLength=20
     maxScrollbarLength=150}}
     {{frost-bunsen-form
-      model=formModel
-      view=formView
-      value=formValue
+      bunsenModel=formModel
+      bunsenView=formView
+      inline=true
```

Comment:
Do we want to allow the consumer to override this as well? I.e. make it come from the properties of the `frost-modal-input` and just default it to `true` (similar to how you handled `showAllErrors`?)